enterprise/backend/src/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions.clj

```diff
@@ -23,8 +23,9 @@
    [metabase.permissions.util :as perms.u]
    [metabase.public-settings.premium-features :refer [defenterprise]]
    [metabase.query-processor.error-type :as qp.error-type]
-   [metabase.query-processor.middleware.fetch-source-query
-    :as fetch-source-query]
+   #_{:clj-kondo/ignore [:deprecated-namespace]}
+   [metabase.query-processor.middleware.fetch-source-query-legacy
+    :as fetch-source-query-legacy]
```

Comment:
I ported the `fetch-source-query` middleware, however a few other pieces of middleware depended on functions there. Instead of having to port all of that stuff too before I'm completely ready, I just copied the old implementation to a new namespace called `fetch-source-query-legacy` for now. Once we port these next pieces of middleware we can remove the copy.